yktrace/src/tir.rs

```diff
@@ -179,6 +181,20 @@ impl<'a> TirTrace<'a> {
                         // The following statements are specific to TIR and cannot appear in SIR.
                         Statement::Call(..) | Statement::StorageDead(_) => unreachable!()
                     };
+
+                    // In TIR, stores to local number zero are always to the return value of the
+                    // #[interp_step] function. We know this is unit so we can ignore it.
```

Comment:
Maybe an assert that the return type is unit?

Reply:
Fixed in 6ce00f5, thanks.